orchestration-shared/src/main/java/uk/gov/di/orchestration/shared/api/FrontEndPages.java

```diff
@@ -0,0 +1,86 @@
+package uk.gov.di.orchestration.shared.api;
+
+import com.nimbusds.oauth2.sdk.ErrorObject;
+import com.nimbusds.openid.connect.sdk.Prompt;
+import uk.gov.di.orchestration.shared.services.ConfigurationService;
+
+import java.net.URI;
+import java.util.HashMap;
+import java.util.Optional;
+
+import static uk.gov.di.orchestration.shared.helpers.ConstructUriHelper.buildURI;
+
+public class FrontEndPages {
+
+    private final URI frontEndBaseUri;
+
+    private static final String PRIVACY_NOTICE = "privacy-notice";
+    private static final String TERMS_OF_SERVICE = "terms-and-conditions";
+    private static final String IPV_CALLBACK = "ipv-callback";
+    private static final String AUTHORIZE = "authorize";
+    private static final String ACCOUNT_BLOCKED = "unavailable-permanent";
+    private static final String ACCOUNT_SUSPENDED = "unavailable-temporary";
+    private static final String DEFAULT_LOGOUT = "signed-out";
+    private static final String ERROR = "error";
+    private static final String ERROR_IPV_CALLBACK = "ipv-callback-session-expiry-error";
+    private static final String PROMPT_PARAMETER_KEY = "prompt";
+    private static final String ERROR_CODE_PARAMETER_KEY = "error_code";
+    private static final String ERROR_DESCRIPTION_PARAMETER_KEY = "error_description";
+    private static final String GOOGLE_ANALYTICS_QUERY_PARAMETER_KEY = "result";
```

Comment:
Would it be more readable to inline all of these? I think they're all used exactly once

Reply:
Fair enough. Will do.